src/renderer/item.store.ts

```diff
@@ -28,8 +28,8 @@ export abstract class ItemStore<T extends ItemObject = ItemObject> {
   }
 
   @action
-  protected sortItems(items: T[] = this.items, sorting?: ((item: T) => any)[], order?: "asc" | "desc"): T[] {
-    return orderBy(items, sorting || this.defaultSorting, order);
+  protected sortItems(items: T[] = this.items, sorting: ((item: T) => any)[] = [this.defaultSorting], order?: "asc" | "desc"): T[] {
```

Comment:
Either way this is unreadable to me. Modern JavaScript/Typescript is for the young.

Reply:
I'll had some doc comments to help clear this up.